shell.nix

```diff
@@ -1,4 +1,10 @@
-with import <nixpkgs> {};
+with import (builtins.fetchGit rec {
+  name = "nixpkgs-19.09-${rev}";
+  url = https://github.com/nixos/nixpkgs;
+  ref = "nixos-19.09";
+  # git ls-remote https://github.com/nixos/nixpkgs-channels nixos-19.09
+  rev = "9f453eb97ffe261ff93136757cd08b522fac83b7";
+}) {};
```

Comment:
I usually use [this repo](https://github.com/NixOS/nixpkgs-channels), since it's only updated when all Hydra tests have passed, whereas the nixpkgs repo's branches only receive ofBorg tests. At least AFAIU.

More info [here](https://nixos.wiki/wiki/FAQ#What_are_channels_and_how_do_they_get_updated.3F).

Reply:
`nixpkgs-channels` is *huge* and takes way longer to download than just `nixpkgs`, so I use the base `nixpkgs` repo and pull the commit from `nixpkgs-channels` only. Happy to change if you feel strongly about it.

Reply:
Looks great, hadn't noticed you were cross-referencing the commit from nixpkgs-channels.